docker-compose.yml

```diff
@@ -95,8 +95,11 @@ services:
       - frontend
     working_dir: /src
     image: node:20.13
-    command: >
-      /bin/sh -c 'yarn install && yarn storybook & yarn workspace mit-open watch:docker'
```

Comment:
I do not understand why, but it seems like the `yarn storybook & yarn workspace mit-open watch:docker` were running BEFORE install finished, which was throwing an error **if `node_modules` was empty**.

I didn't catch it in https://github.com/mitodl/mit-open/pull/936 because my node_modules wasn't empty.

Reply:
The error was because `/bin/sh` is left-associative with `&&` and `&`.

```sh
/bin/sh -c 'sleep 5 && echo A & echo B'
```
prints B immediately, then does sleep and prints A.